js/file_provider/file_provider.ajax.js

```diff
@@ -0,0 +1,76 @@
+import ajax from "../core/utils/ajax";
+import { Deferred } from "../core/utils/deferred";
+import { FileProvider } from "./file_provider";
+import ArrayFileProvider from "./file_provider.array";
+
+var AjaxFileProvider = FileProvider.inherit({
```

Comment:
Use ES6 class:
`class AjaxFileProvider extends FileProvider`

Reply:
Done.  All code changed according to ES6 standard.